lib/versioning/semver-coerced/index.ts

```diff
@@ -81,7 +84,8 @@ function isSingleVersion(version: string): string | boolean | null {
 }
 
 // If this is left as an alias, inputs like "17.04.0" throw errors
-export const isVersion = (input: string): string | boolean => isValid(input);
+export const isVersion = (input: string): string | boolean | null =>
```

Comment:
@rarkins We should change the `VersioningApi` interface to always return bool for `isCompatible`, `isSingleVersion`, `isValid` and `isVersion`

Reply:
Do we need to do so before continuing with these versioning checks?

Reply:
i think it's independend

Reply:
https://github.com/renovatebot/renovate/blob/1445fb07bf1f530bb635e66b3521a1966ec9712f/lib/versioning/ubuntu/index.ts#L14-L19

Is the type-check necessary when the argument type is already declared?